src/Traits/ClientEndpointsTrait.php

```diff
@@ -1404,20 +1434,20 @@ public function rankEval(array $params = [])
 	 * @see https://www.elastic.co/guide/en/elasticsearch/reference/master/docs-reindex.html
 	 *
 	 * @param array{
-	 *     refresh: boolean, // Should the affected indexes be refreshed?
-	 *     timeout: time, // Time each individual bulk request should wait for shards that are unavailable.
-	 *     wait_for_active_shards: string, // Sets the number of shard copies that must be active before proceeding with the reindex operation. Defaults to 1, meaning the primary shard only. Set to `all` for all shard copies, otherwise set to any non-negative value less than or equal to the total number of copies for the shard (number of replicas + 1)
-	 *     wait_for_completion: boolean, // Should the request should block until the reindex is complete.
-	 *     requests_per_second: number, // The throttle to set on this request in sub-requests per second. -1 means no throttle.
-	 *     scroll: time, // Control how long to keep the search context alive
-	 *     slices: number|string, // The number of slices this task should be divided into. Defaults to 1, meaning the task isn't sliced into subtasks. Can be set to `auto`.
-	 *     max_docs: number, // Maximum number of documents to process (default: all documents)
-	 *     pretty: boolean, // Pretty format the returned JSON response. (DEFAULT: false)
-	 *     human: boolean, // Return human readable values for statistics. (DEFAULT: true)
-	 *     error_trace: boolean, // Include the stack trace of returned errors. (DEFAULT: false)
-	 *     source: string, // The URL-encoded request definition. Useful for libraries that do not accept a request body for non-POST requests.
-	 *     filter_path: list, // A comma-separated list of filters used to reduce the response.
-	 *     body: array, // (REQUIRED) The search definition using the Query DSL and the prototype for the index request.
+	 *     refresh?: bool, // Should the affected indexes be refreshed?
+	 *     timeout?: int|string, // Time each individual bulk request should wait for shards that are unavailable.
+	 *     wait_for_active_shards?: string, // Sets the number of shard copies that must be active before proceeding with the reindex operation. Defaults to 1, meaning the primary shard only. Set to `all` for all shard copies, otherwise set to any non-negative value less than or equal to the total number of copies for the shard (number of replicas + 1)
+	 *     wait_for_completion?: bool, // Should the request should block until the reindex is complete.
+	 *     requests_per_second?: float, // The throttle to set on this request in sub-requests per second. -1 means no throttle.
+	 *     scroll?: int|string, // Control how long to keep the search context alive
+	 *     slices?: float|number|string, // The number of slices this task should be divided into. Defaults to 1, meaning the task isn't sliced into subtasks. Can be set to `auto`.
```

Comment:
This one still has the invalid `number` type as part of it's signature.

It's also valid to have the literal string in the PHPDoc type so the ideal type would be `int|'audo'`